src/main/java/net/coderbot/iris/shaderpack/ShaderPack.java

```diff
@@ -89,6 +98,35 @@ private static String readFile(Path path) throws IOException {
 		}
 	}
 
+	private Map<String, Map<String, String>> parseLangEntries(Path root) throws IOException {
+		Path langFolderPath = root.resolve("lang");
+		Map<String, Map<String, String>> allLanguagesMap = new HashMap<>();
+
+		if (!Files.exists(langFolderPath)) {
+			return allLanguagesMap;
+		}
+			Files.walk(langFolderPath, 1).filter(path -> !Files.isDirectory(path)).forEach(path -> {
+
+				Map<String, String> currentLanguageMap = Maps.newHashMap();
+				String currentFileName = path.getFileName().toString().toLowerCase();
+
+				String currentLangCode = currentFileName.substring(0, currentFileName.lastIndexOf("."));
+				Properties properties = new Properties();
+
+				try {
+					properties.load(Files.newInputStream(path));
+				} catch (IOException e) {
+					Iris.logger.error("Error while parsing languages for shaderpacks!", e);
```

Comment:
Maybe include the file path in the error message? That would probably be helpful for shaderpack authors